gradle-jdks-integration-tests/src/jdkIntegrationTest/resources/ignore-certs-curl-wget.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -e
+
+if command -v curl > /dev/null 2>&1; then
+  function curl () { command curl -k "$@"; }
+  export curl
+fi
```

Comment:
I was wondering why this couldn't just be written `alias curl='curl -k'` but I think this is so we don't set `curl` so it looks like it's installed? Might be worth a comment.

Reply:
aliases are not supported in Bourne shells unfortunately. This is the option that worked for all shell types.